app/registries/menu/loader.rb

```diff
@@ -94,6 +94,19 @@ def self.load
       Manager.map :labs_menu do |menu|
         menu.sub_menu :lab_features_menu, :caption => N_('Lab Features'), :icon => 'fa fa-flask'
       end
+
+      if Rails.env.development?
+        Manager.map :devel_menu do |menu|
+          menu.sub_menu :devel_tools, :caption => 'Develop tools', :icon => 'pficon pficon-maintenance' do
```

Comment:
I would choose a single word as a caption, e.g. `Toolbox`.

Reply:
I changed it to `Toolbox`